repository/soundboard.go

```diff
@@ -0,0 +1,15 @@
+package repository
+
+import (
+	"github.com/gofrs/uuid"
+
+	"github.com/traPtitech/traQ/model"
+)
+
+type SoundboardRepository interface {
+	CreateSoundboardItem(soundID uuid.UUID, soundName string, stampID *uuid.UUID, creatorID uuid.UUID) error
+	GetAllSoundboardItems() ([]*model.SoundboardItem, error)
+	GetSoundboardByCreatorID(creatorID uuid.UUID) ([]*model.SoundboardItem, error)
+	UpdateSoundboardCreatorID(soundID uuid.UUID, creatorID uuid.UUID) error
+	DeleteSoundboardItem(soundID uuid.UUID) error
+}
```

Comment:
他の部分だと以下のように書かれていることが多そうなので、引数の多めなメソッドの引数部分はまとめちゃっていいと思います
同時に、各メソッドに説明もコメントによる説明もつけられるとさらに読みやすいコードになるかと思います！
```suggestion
type CreateSoundboardItemArgs struct {
	soundID uuid.UUID
	soundName string
	stampID *uuid.UUID
	creatorID uuid.UUID
}

type SoundboardRepository interface {
	// CreateSoundboardItem スタンプを作成します
	//
	// ここに説明文
	CreateSoundboardItem(args CreateSoundboardItemArgs) error
	GetAllSoundboardItems() ([]*model.SoundboardItem, error)
	GetSoundboardByCreatorID(creatorID uuid.UUID) ([]*model.SoundboardItem, error)
	UpdateSoundboardCreatorID(soundID uuid.UUID, creatorID uuid.UUID) error
	DeleteSoundboardItem(soundID uuid.UUID) error
}
```